content/authors/leonardoce/index.md

```diff
@@ -0,0 +1,20 @@
+---
+name:  Leonardo Cecchi
+avatar: leonardoce-gh.jpeg
+github: leonardoce
+---
+
+I’ve been in computer science from the beginning, and programming has always
+been my obsession. My passion began when I was 6 when my father taught me the
+rudiments of C programming.
+
+I’ve been working in 2ndQuadrant since 2016 as a developer and a PostgreSQL
+consultant. PostgreSQL has always been my choice when working with a DBMS. I’ve
+been using it since 1998. 
+
+EDB acquired 2ndQuadrant in 2020 and CloudNative-PG, the operator me and my team
+has been working on since 2019, was finally released as OSS in 2022.
```

Comment:
```suggestion
EDB acquired 2ndQuadrant in 2020, and CloudNativePG, the operator I
have been working on with my team since 2019, was finally released as OSS in 2022.
```